### PR TITLE
Deprecated the SYMFONY__ variables

### LINKED ISSUE #7695: [DI] Deprecate the special SYMFONY__ environment variables
See https://github.com/symfony/symfony/pull/21889

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7695.

### CODE REVIEW

File: configuration/external_parameters.rst [hunk 1]
```diff
@@ -114,6 +114,12 @@ of the following:
 
 .. tip::
 
+    .. versionadded:: 3.3
+        The support of the special ``SYMFONY__`` environment variables was
+        deprecated in Symfony 3.3 and it will be removed in 4.0. Instead of
+        using those variables, define regular environment variables and get
+        their values using the ``%env(...)%`` syntax in your config files.
```
Comment (HeahDude) [inline configuration/external_parameters.rst]: "their value"?

Comment (javiereguiluz) [inline configuration/external_parameters.rst]: I'm confused about this. I don't know which is the best option here.

Comment (HeahDude) [inline configuration/external_parameters.rst]: I have no doubt in French we definitely mean "each" when using "their". To me in this case, each environment variable has one value and not many, I've found many discussions on the web, looks like this confusion is global, see http://english.stackexchange.com/questions/131430/should-a-noun-after-the-determiner-their-take-in-the-form-of-singular-or-of-pl for a reference.

Comment (xabbuh) [inline configuration/external_parameters.rst]: "their values" sounds right to me

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request merged**